std/node/global.ts

```diff
@@ -24,4 +25,18 @@ Object.defineProperty(globalThis, "Buffer", {
   configurable: true,
 });
 
+Object.defineProperty(globalThis, "setImmediate", {
+  value: timers.setImmediate,
+  enumerable: false,
```

Comment:
```suggestion
  enumerable: true,
```
Ditto on line 37.

Reply:
Yep, you're right - I mistakenly assumed that it would be the same as the other globals.